src/components/Box/Box.tsx

```diff
@@ -24,13 +24,13 @@ interface BoxProps
     extends SpaceProps,
         LayoutProps,
         PositionProps,
-        ColorProps,
+        Omit<ColorProps, 'color'>, // HACK: avoid collision of `color` prop
```

Comment:
what else do we have inside colorProps in styled-system?

Reply:
bg color and opacity